docs/LOGS_PRIVACY.md

```diff
@@ -0,0 +1,45 @@
+# Logs Privacy in rbuilder
```

Comment:
Imo, ideally the docs have a date in the filename (and/or in the text), for additional context. i.e. `20240911_logs-privacy.md`. Not a strong opinion, and up to you.